src/mem3/src/mem3_sync_event_listener.erl

```diff
@@ -301,30 +301,19 @@ get_state(Pid) ->
     Pid ! {get_state, Ref, self()},
     receive
         {Ref, State} -> State
-    after 10 ->
+    after 500 ->
         timeout
     end.
 
 
-wait_state_frequency(Pid, Val) ->
+wait_state(Pid, Field, Val) when is_pid(Pid), is_integer(Field) ->
+    
```

Comment:
Extra space (& tab) here?

Reply:
Thanks for noticing. I have no idea why I inserted it there. I'll fix it